### ISSUE TITLE
parallel_apply(..., axis=0) return duplicated rows.

### ISSUE THREAD

**Issue by miettal** [nalepae/pandarallel#74]
this behavior is different from pandas.

**Comment by nalepae** [comment]
Could you please provide me more specific details?
(OS, Pandas version, Pandarallel version, "faulty" dataframe example, step to reproduce this issue...)

Thanks !

**Comment by gaussleescorpio** [comment]
I have the same issues as well.  If I use 5 workers, df.paralell_apply will return df with 5 times lengthier than original one

**Comment by gaussleescorpio** [comment]
close_price = sub_d.set_index(["code", "date"])["close"].unstack(level=0).to_pandas()
    close_price = close_price.replace("", np.nan)
    s = close_price.parallel_apply(calculate_rsi_indicators)
print(close_price.shape, s.shape)

The answer is :

(242, 2267) (484, 2267)

shape changed. Even a simple case, it does not work.

**Comment by gaussleescorpio** [comment]
ubuntu 16.04, pandas 0.25.1

**Comment by gaussleescorpio** [comment]
Okay. I will reply by myself. If you still wanna avoid this bug, you have to transpose your dataframe S.

S.T.parallel_apply(func, axis=1).T

**Comment by inazr** [comment]
The number of rows is multiplied by the number of nb_workers. 

|  | A | B |
|--|--|--|
| 0| 9 | 4 |
| 1| 9 | 4 |
| 2| 9 | 4 |

with 2 workers and -> df.parallel_apply(sqrt, axis=0)

|  | A | B |
|--|--|--|
| 0| 3 | NaN |
| 1| 3 | NaN |
| 2| 3 | NaN |
| 0| NaN | 2 |
| 1| NaN | 2 |
| 2| NaN | 2 |

**Comment by farridav** [comment]
Is this still an issue? If not, can it be closed ?

When testing this i get normal results

```
In [1]: pd.DataFrame(data={'a': [1,2,3]}).apply(lambda x: x).shape == pd.DataFrame(data={'a': [1,2,3]}).parallel_apply(lambda x: x).shape
Out[1]: True
```

**Comment by Anthony2cc** [comment]
Same Problem in macos 11.1 with latest pandas and pandarallel in python 3.8

**Comment by Anthony2cc** [comment]
> Is this still an issue? If not, can it be closed ?
> 
> When testing this i get normal results
> 
> ```
> In [1]: pd.DataFrame(data={'a': [1,2,3]}).apply(lambda x: x).shape == pd.DataFrame(data={'a': [1,2,3]}).parallel_apply(lambda x: x).shape
> Out[1]: True
> ```

ur case is one dimension, try a multiple one with nb_workers>1

**Comment by nalepae** [comment]
Fixed in `pandarallel v1.6.2`
